frontend/src/app/models/specification.ts

```diff
@@ -5,6 +5,22 @@ export enum ApiLanguage {
   GraphQL = 'GRAPHQL',
 }
 
+export enum ReleaseType {
+  Release = 'RELEASE',
+  Prerelease = 'PRERELEASE',
+  Snapshot = 'SNAPSHOT',
+}
+
+export function inferReleaseType(version: string): ReleaseType {
```

Comment:
Can't we send this information from the backend isntead of inferring it in the frontend?